gpytorch/kernels/kernel.py

```diff
@@ -193,29 +192,28 @@ def __pdist_dist(self, x1):
 
         return res
 
-    def __slow_sq_dist(self, x1, x2, diag, x1_eq_x2):
+    @torch.jit.script
```

Comment:
Hmm so I'm not familiar with this - if you define a `@torch.jit.script` function on a class, it doesn't take the standard `self` arg? (makes sense b/c it's scripted and can't work with arbitrary python objects, but still somewhat confusing). 

See my comment below how this could be re-used.

Reply:
Yeah, that seems to be the case -- things break if it takes self.